### ISSUE TITLE
Unknown encoding cp437

### ISSUE THREAD

**Issue by vogt31337** [slimtoolkit/slim#689]
Expected Behavior
=================

I've built a python web server application which reads zip and xml files, modifies and writes them. Everything works fine until a slimified version is created.

---
Actual Behavior
=================

Application crashes with - ERROR - unknown encoding: cp437
Without slimification it works fine.

Maybe slim did remove the codepage. This behaviour is nowhere mentioned or documented.

---
Specifications
=================
  - Version: latest
  - Platform: kubernetes

**Comment by kcq** [comment]
@vogt31337 sounds like you might need to define custom http probes for your endpoints. You can use the `--http-probe-cmd` and `--http-probe-cmd-file` flags for it. With the most basic setup you just specify the server path for the endpoint. Example: `--http-probe-cmd /my/custom/endpoint`


Additional info for others when you have non-server CLI apps:

For a CLI app you need to configure the slim command to run the app in the container. There are a few options for that. 

You can use the `--exec` flag to run a shell command in the target container (e.g., your app with sample params). 

You can also use the `--entrypoint` and/or `--cmd` slim command flags (similar to how the ENTRYPOINT and CMD instructions in Dockerfiles are used) to point to your CLI app and invoke it with specific parameters. If you need file data to references when you are invoking your CLI app and those files don't exist in the container you can mount those files using the `--mount` flag.

**Comment by kcq** [comment]
@vogt31337 are you using FastAPI for your python server app by any chance? If so, it exposes an openapi spec that can be used to generate additional interactions with the app.

**Comment by vogt31337** [comment]
Yeah I am using fastapi, but again that ist Not the Problem (the swagger Style documentation works). Reading an XML File from a mounted storage fails with this encoding Error.
And in the fat Image IT doesn't fail.
